### PR TITLE
contribution UI as per figma design

### PR DESCRIPTION
**Pull request by RohitKuwar**
@sumitd94 I have made changes according to the new Figma Design. Please review the PR.

Desktop mode
Screenshot (759)

Mobile Devices
Screenshot (760)

### CODE REVIEW

File: components/member-profile/contribution/index.js [hunk 1]
```diff
@@ -45,43 +34,31 @@ const Contribution = ({ contribution, fullName, imageLink }) => {
...
+        <div className={classNames.leftSection}>
+          <h3 className={classNames.featureTitle}>{isTaskAvailable ? title : prList[0].title}</h3>
+          <div className={classNames.completedData}>
+            {completedText}
+            <p className={classNames.completedDate}>{completedDate}</p>
+          </div>
+          <div className={classNames.featureLiveOnText}>{featureLiveOnText}</div>
+        </div>
+        <div className={classNames.rightSection}>
+          <div className={classNames.prLink}>{renderPRLinks(prList)}</div>
```
Comment (ankushdharkar) [inline components/member-profile/contribution/index.js:55]: Very Nice!

File: components/member-profile/contribution/index.js [hunk 2]
```diff
@@ -9,22 +8,12 @@ const renderPRLinks = (prList) =>
...
-  const [showMoreContent, setShowMoreContent] = useState(true);
-
-  const showMoreContentHandler = () => {
-    setShowMoreContent((prevstate) => !prevstate);
-  };
-
-  const showText = showMoreContent ? 'Show less' : 'Show more';
-  const showMoreContentClass = showMoreContent ? classNames.showContent : classNames.hideContent;
-
   const isTaskAvailable = title ? true : false;
```
Comment (ankushdharkar) [inline components/member-profile/contribution/index.js:16]: Nit: Should this be `isTitleAvailable`?

File: components/member-profile/contribution/index.js [hunk 3]
```diff
@@ -45,43 +34,31 @@ const Contribution = ({ contribution, fullName, imageLink }) => {
...
-          </p>
-          <p className={classNames.featureLink}>
-            <a href={featureUrl} target="_blank" rel="noreferrer">
-              Click here to checkout this feature
-            </a>
-          </p>
+    <div className={classNames.mainSection}>
+      <div className={classNames.contributionContainer}>
+        <div className={classNames.leftSection}>
+          <h3 className={classNames.featureTitle}>{isTaskAvailable ? title : prList[0].title}</h3>
```
Comment (ankushdharkar) [inline components/member-profile/contribution/index.js:47]: Nit: Can we have this logic before the JSX?
```js
const featureTitle = isTaskAvailable ? title : prList[0].title;
```

Follow up question: Will `prList` have at least one PR always?

Comment (RohitKuwar) [reply components/member-profile/contribution/index.js:47]: done

File: components/member-profile/contribution/index.js (hunk 1 above)
Comment (RohitKuwar) [reply components/member-profile/contribution/index.js:55]: thank you

File: components/member-profile/contribution/index.js (hunk 2 above)
Comment (RohitKuwar) [reply components/member-profile/contribution/index.js]: done

**Review by sumitd94 — APPROVED** [review]
Awesome work 🥇

**Comment by RohitKuwar** [comment]
@ankushdharkar @sumitd94 Please review the PR. I have made the new changes.

Desktop View
Screenshot (777)

Mobile view
Screenshot (779)

**Comment by RohitKuwar** [comment]
> 
> 
> Awesome work 🥇

Thank you.

**Pull request merged**